cms/templates/js/verification-access-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title"><%= gettext('Verification Checkpoint') %></h3>
+<div class="modal-section-content verification-access">
+  <ul class="list-fields list-input">
+    <li class="field field-checkbox checkbox-cosmetic">
+      <input
+        type="checkbox"
+        id="verification-access-checkbox"
+        class="input input-checkbox"
+        name="verification-access"
+        value=""
+        <% if (hasVerificationRestrictions) { %> checked="checked" <% } %>
+      >
+      <label class="label" for="verification-access-checkbox">
+        <i class="icon fa fa-check-square-o input-checkbox-checked"></i>
+        <i class="icon fa fa-square-o input-checkbox-unchecked"></i>
+
+        <select id="verification-partition-select">
+          <% for (var i = 0; i < verificationPartitions.length; i++) {
+            partition=verificationPartitions[i];
+          %>
+            <option
+              value="<%- partition.id %>"
+              <% if (partition.selected) { %> selected <% } %>
+            ><%- partition.name %></option>
+          <% } %>
+        </select>
+      </label>
+      <span class="note"><%= gettext("Learners who require verification must pass the selected checkpoint to see the content in this unit. Learners who do not require verification see this content by default.") %></span>
```

Comment:
@cptvitamin This is the most recent version of the other dialog in this PR.  It looks like this:
![image](https://cloud.githubusercontent.com/assets/2948394/9342546/467d078e-45b1-11e5-94d4-77f7763cb675.png)